### PR TITLE
Rename all internal references from 'ashlar' to 'grout'

### PR DESCRIPTION
**Pull request by jeancochrane**
# Overview

This PR renames all references to `ashlar` to `grout` (including variants like `Ashlar` and `ASHLAR`). It also updates `setup.py` and `.travis.yml` to make sure that the new name is reflected in packaging and CI. 

## Notes

- Not much interesting code in here, since this is largely just s/find/replace/g.

## Testing

- Travis will run unit tests for all supported versions of Python and Django. To run the tests locally, use the `scripts` module:

```bash
# Update containers
./scripts/update

# Run tests
./scripts/test
```

- To confirm that all references to "Ashlar" have been purged, you can grep for `ashlar`, `Ashlar`, and `ASHLAR`.

Closes https://github.com/azavea/grout-2018-fellowship/issues/30.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,40 +1,40 @@
 language: python
 python:
-  - '2.7'
-  - '3.4'
-  - '3.5'
-  - '3.6'
+- '2.7'
```
Comment (ddohler) [inline .travis.yml]: Was this intentional? I think the old formatting looked correct.

File: .travis.yml [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
+    server: https://test.pypi.org/legacy/
     password:
-      secure: iP4WfDokE1JImVAw4udpnc5ZRvyXsMM4xLGWFNJ8dX13SOP5eCGXhghhKRIKOW84t1upb396+FYZX0IXO+kWlYa1Lzvry3x3fXLBQryRZtQ7Pf97e/EYn2IzjngxQ7mwg6qe2MaH4GS2cGMs9/6UYaXwqr0Ro2d0zBEQ4yV1Z8U=
+      secure: gVivn0yuENFhivdr7ZbE6WQc8SNa9ocWph2UWdfhPLXxpzGivjTJmHNHyU+8VyXqD25rxLDm6KYFOgxKKzdjBsNvle9Jw4DvJkVnO06RJiYJK8SYAE5HElowRpBHu1UNTjh86U1YI6StRsvJGB8zogAshHhhQmpGsmNICkuQc1VCJJHtTOYVLsIVJodAAc98BfR2u+4guz6h3MVjWgIUhk+kqVMv9oq87ryPlWxu5A2w3ZP+abLFdZ2cXLNsx8ZIzmAEYgCOcJkX17Q4mTNWX7kfOy7hYT3TlQhAK90i5qvIlkZmGy6NCuLV3ToxEyGvY4tTqRs4WvWgf3+4P2DblCVbMwednnNunS1cu64FD8Jll3KGV83vWvnJRCF4LgI80E/WN+1T6EbQU71NLb+JDyZsT0UiCFAMoGu+jdg4iqKP3Y5GyOd7ozlhXfoCl+uNHmOaO/idtt2FVSL3+vSMbu1HWsXST2sX1ojOM9OVRoiebxPVzCUDYyc/Z1OMHfp/S8xbGFhjhPwFXTBJ3xzUzCyWUme5oeJhKJuQPXBPXr2yyaUyVIobmBaJnhXPI/1N666tLihYYpNzgs7l5ZNllA0By3KQ9kZjN/WcOlELjnOVlUUNOI9b9b0FvjvFjKrKt2No4VXihRN0VeamZolpO02Kt+mnvcPRpVSKKUqEtGA=
     on:
       distributions: sdist bdist_wheel
-      repo: azavea/ashlar
-      branch: develop
+      repo: azavea/grout
+      branch: deploy
```
Comment (ddohler) [inline .travis.yml]: This should probably stay as `develop` to match git-flow. https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

**Review by ddohler — COMMENTED** [review]
I had a couple questions but this looks good otherwise -- if Travis is happy, I'm happy.

File: .travis.yml (hunk 1 above)
Comment (jeancochrane) [reply .travis.yml]: Hmm, I think this has happened to me before when running a travis-cli command on an existing .travis.yml file (in this case, I ran `travis setup pypi`). For some reason it reformats the whole file? In any case I agree that the previous formatting was nicer, so I'll change it back.

File: .travis.yml (hunk 2 above)
Comment (jeancochrane) [reply .travis.yml]: Oops, yes, good catch!

**Review by ddohler — APPROVED** [review]

**Comment by jeancochrane** [comment]
Looks like django-filter [dropped support for Python 2.7 three hours ago](https://github.com/carltongibson/django-filter/releases/tag/2.0.0), which is making Travis fail. I'll need to pin that dependency in setup.py to get this to pass again.

**Pull request merged**